source/extensions/filters/http/local_ratelimit/local_ratelimit.cc

```diff
@@ -109,6 +118,31 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::RequestHeaderMap& headers,
   return Http::FilterHeadersStatus::StopIteration;
 }
 
+bool Filter::requestAllowed(absl::Span<const RateLimit::LocalDescriptor> request_descriptors) {
+  return getRateLimiter().requestAllowed(request_descriptors);
+}
+
+const Filters::Common::LocalRateLimit::LocalRateLimiterImpl& Filter::getRateLimiter() {
+  const auto* config = getConfig();
```

Comment:
move this closer to where it's used, e.g.: just before `decoder_callbacks_->streamInfo().filterState()->setData()`

Reply:
Moved it back up a couple of lines to add an assertion.